samples/nrf_desktop/configuration/nrf52_pca20037/flash.overlay

```diff
@@ -0,0 +1 @@
+/* Currently no dedicated partition sizes. */
```

Comment:
AFAIK flash.overlay is not used any more.

Reply:
leftover after rebase. thanks